docs/configuration.md

```diff
@@ -55,8 +55,6 @@ The following settings determine how different types should be evaluated.
 
 - <a name="strictListInference"></a> **strictListInference** [boolean]: When inferring the type of a list, use strict type assumptions. For example, the expression `[1, 'a', 3.4]` could be inferred to be of type `list[Any]` or `list[int | str | float]`. If this setting is true, it will use the latter (stricter) type. The default value for this setting is `false`.
 
-- <a name="strictListInference"></a> **strictListInference** [boolean]: When inferring the type of a list, use strict type assumptions. For example, the expression `[1, 'a', 3.4]` could be inferred to be of type `list[Any]` or `list[int | str | float]`. If this setting is true, it will use the latter (stricter) type.
```

Comment:
can we keep the other one instead, since i consider the "The default value for this setting is `false`" part redundant and therefore prone to mistakes sicne the defaults are all listed in the table below

Reply:
You are right, this is the only place where there is an indication of the default rule.